src/Model/EventMapper.php

```diff
@@ -1108,10 +1110,11 @@ public function approve($event_id, $reviewing_user_id)
      *
      * @param  integer $event_id
      * @param  integer $reviewing_user_id The user who rejected the event
+     * @param  string $reason_Details for rejection
      *
      * @return boolean
      */
-    public function reject($event_id, $reviewing_user_id)
+    public function reject($event_id, $reviewing_user_id, $reason)
```

Comment:
What do you think about adding a type declaration here?

```suggestion
    public function reject($event_id, $reviewing_user_id, string $reason)
```